.golangci.yaml

```diff
@@ -7,7 +7,9 @@ linters-settings:
     values:
       const:
         COMPANY: Cofide Limited
+      regexp:
+        VALID_YEAR: 202[4-9]|20[3-9][0-9]|2[1-9][0-9][0-9]
```

Comment:
Why not
```
2[0-9][0-9][0-9]
```
?

Reply:
I suppose yours doesn't allow years before 2024.

Reply:
Yeah I just wanted to make it 2024 and onwards.